src/researches/buildKeywordForms.js

```diff
@@ -0,0 +1,66 @@
+const getForms = require( "../morphology/english/getForms.js" );
+const getWords = require( "../stringProcessing/getWords.js" );
+const getLanguage = require( "../helpers/getLanguage.js" );
+const getFunctionWords = require( "../helpers/getFunctionWords.js" )();
+
+const includes = require( "lodash/includes" );
+const filter = require( "lodash/filter" );
+const escapeRegExp = require( "lodash/escapeRegExp" );
+
+/**
+ * Analyses the focus keyword string. Checks if morphology is requested or if the user wants to match exact string.
+ * If morphology is required the module builds all wordforms for all content words (prepositions, articles, conjunctions).
+ *
+ * @param {Object} paper The paper to analyze keyword forms for.
+ *
+ * @returns {array} Array of all forms to be searched for keyword-based assessments.
+ */
+
+module.exports = function( paper ) {
+	let keyword = paper.getKeyword();
+
+	// If the keyword is embedded in double quotation marks no morphological analysis is required.
+	// Return keyword itself, without outer-most quotation marks.
+	if ( keyword[ 0 ] === "\"" && keyword[ keyword.length - 1 ] === "\"" ) {
```

Comment:
Concerning the exact match functionality (i.e., using quotation marks): I think it makes sense to use quotation marks for this in the prototype, but when we release this as a feature there should probably be a different interface solution. This could, for example, be a checkbox that users can check to switch on exact matching functionality.

Reply:
I think either way is fine. Quotation marks is how you search on Google, so I think it is valid, too.